src/overlays/actors/ovl_En_Ge1/z_en_ge1.c

```diff
@@ -27,7 +27,7 @@ void EnGe1_Destroy(Actor* thisx, PlayState* play);
 void EnGe1_Update(Actor* thisx, PlayState* play);
 void EnGe1_Draw(Actor* thisx, PlayState* play);
 
-s32 EnGe1_CheckCarpentersFreed(void);
+s32 EnGe1_CheckAllCarpentersRescued(void);
```

Comment:
imo dont even need "check" in this case, "if all carpenters resuced" reads nicely

Reply:
same would apply to ge2 if you do change it

Reply:
eh, I think it's better as-is. `AllCarpentersRescued` in a vacuum reads more towards being the event of having rescued all carpenters.